.github/workflows/TagBot.yml

```diff
@@ -4,6 +4,22 @@ on:
     types:
       - created
   workflow_dispatch:
+    inputs:
+      lookback:
+        default: "3"
+permissions:
+  actions: read
+  checks: read
+  contents: write
+  deployments: read
+  issues: read
+  discussions: read
+  packages: read
+  pages: read
+  pull-requests: read
+  repository-projects: read
+  security-events: read
+  statuses: read
```

Comment:
Why does this one need to be different from the other repos?

Reply:
This isn't different, it's just explicitly mentioned while the TagBot workflows in other repos are likely not up-to-date.
This is the recommended list of permissions, as per [TagBot's README.md](https://github.com/thazhemadam/TagBot/blob/16a1824512433d6aa510b6f43a9f00877510c6b9/README.md#setup).